doc/api/child_process.md

```diff
@@ -789,9 +789,10 @@ within the child process to close the IPC channel as well.
 added: v0.1.90
 -->
 
-* `signal` {String}
+* `signal` {String|number} The signal to send, either as a string or number.
```

Comment:
Fixed! That was actually a c+p from process.md, so I'll add another commit to fix that file. Or should I open a different PR for that?